cmake/RuntimeChecks.cmake

```diff
@@ -106,9 +106,17 @@ elseif(RUNTIME_CHECK STREQUAL "asan")
     message(FATAL_ERROR "libubsan not installed - address sanitizer not available")
   endif(UBSAN_LIBRARY-NOTFOUND)
   message(STATUS "Runtime memory checker: gcc/clang address sanitizers")
+  option(SANITIZE_3RD_PARTY "Detect leaks in 3rd party libraries used by Dispatch while running tests" OFF)
+  file (COPY "${CMAKE_SOURCE_DIR}/tests/lsan.supp" DESTINATION "${CMAKE_BINARY_DIR}/tests")
+  if (NOT SANITIZE_3RD_PARTY)
```

Comment:
The flag is called SANITIZE_3RD_PARTY but it seems to be specific to libpython2.*.so  and libpython3.*.so
Is Python really third party ?

Reply:
There used to be python, libwebsockets and proton in the .so ignores. Proton is I'd say 2nd party and it is not supposed to leak, so the so should not be suppressed wholesale. The libwebsockets ignore will be unnecessary, so I pulled it out into the lsan.supp directly and I have a plan to fix the leak there. Now only Python remains.

I think I prefer having an option with a generic name. imagine if there were any leaks in cyrus sasl or openssl, those would go under this option. It is possible that by fixing other suppressions, something like this will crop up.